src/full/Agda/Syntax/Builtin.hs

```diff
@@ -81,264 +570,510 @@ builtinNat, builtinSuc, builtinZero, builtinNatPlus, builtinNatMinus,
+builtinAgdaTCM                           = BuiltinAgdaTCM
+builtinAgdaTCMReturn                     = BuiltinAgdaTCMReturn
+builtinAgdaTCMBind                       = BuiltinAgdaTCMBind
+builtinAgdaTCMUnify                      = BuiltinAgdaTCMUnify
+builtinAgdaTCMTypeError                  = BuiltinAgdaTCMTypeError
+builtinAgdaTCMInferType                  = BuiltinAgdaTCMInferType
+builtinAgdaTCMCheckType                  = BuiltinAgdaTCMCheckType
+builtinAgdaTCMNormalise                  = BuiltinAgdaTCMNormalise
+builtinAgdaTCMReduce                     = BuiltinAgdaTCMReduce
+builtinAgdaTCMCatchError                 = BuiltinAgdaTCMCatchError
+builtinAgdaTCMGetContext                 = BuiltinAgdaTCMGetContext
+builtinAgdaTCMExtendContext              = BuiltinAgdaTCMExtendContext
+builtinAgdaTCMInContext                  = BuiltinAgdaTCMInContext
+builtinAgdaTCMFreshName                  = BuiltinAgdaTCMFreshName
+builtinAgdaTCMDeclareDef                 = BuiltinAgdaTCMDeclareDef
+builtinAgdaTCMDeclarePostulate           = BuiltinAgdaTCMDeclarePostulate
+builtinAgdaTCMDeclareData                = BuiltinAgdaTCMDeclareData
+builtinAgdaTCMDefineData                 = BuiltinAgdaTCMDefineData
+builtinAgdaTCMDefineFun                  = BuiltinAgdaTCMDefineFun
+builtinAgdaTCMGetType                    = BuiltinAgdaTCMGetType
+builtinAgdaTCMGetDefinition              = BuiltinAgdaTCMGetDefinition
+builtinAgdaTCMBlockOnMeta                = BuiltinAgdaTCMBlockOnMeta
+builtinAgdaTCMCommit                     = BuiltinAgdaTCMCommit
+builtinAgdaTCMQuoteTerm                  = BuiltinAgdaTCMQuoteTerm
+builtinAgdaTCMUnquoteTerm                = BuiltinAgdaTCMUnquoteTerm
+builtinAgdaTCMQuoteOmegaTerm             = BuiltinAgdaTCMQuoteOmegaTerm
+builtinAgdaTCMIsMacro                    = BuiltinAgdaTCMIsMacro
+builtinAgdaTCMWithNormalisation          = BuiltinAgdaTCMWithNormalisation
+builtinAgdaTCMWithReconstructed          = BuiltinAgdaTCMWithReconstructed
+builtinAgdaTCMWithExpandLast             = BuiltinAgdaTCMWithExpandLast
+builtinAgdaTCMWithReduceDefs             = BuiltinAgdaTCMWithReduceDefs
+builtinAgdaTCMAskNormalisation           = BuiltinAgdaTCMAskNormalisation
+builtinAgdaTCMAskReconstructed           = BuiltinAgdaTCMAskReconstructed
+builtinAgdaTCMAskExpandLast              = BuiltinAgdaTCMAskExpandLast
+builtinAgdaTCMAskReduceDefs              = BuiltinAgdaTCMAskReduceDefs
+builtinAgdaTCMFormatErrorParts           = BuiltinAgdaTCMFormatErrorParts
+builtinAgdaTCMDebugPrint                 = BuiltinAgdaTCMDebugPrint
+builtinAgdaTCMNoConstraints              = BuiltinAgdaTCMNoConstraints
+builtinAgdaTCMRunSpeculative             = BuiltinAgdaTCMRunSpeculative
+builtinAgdaTCMExec                       = BuiltinAgdaTCMExec
+builtinAgdaTCMGetInstances               = BuiltinAgdaTCMGetInstances
+builtinAgdaTCMPragmaForeign              = BuiltinAgdaTCMPragmaForeign
+builtinAgdaTCMPragmaCompile              = BuiltinAgdaTCMPragmaCompile
```

Comment:
These definitions look rather obsolete now.
Maybe you didn't want to touch too many files (so kept these as backwards-compatibility)?